SymbolTableTest/IntegrationTest.cs

```diff
@@ -0,0 +1,110 @@
+using System;
+using Xunit;
+using GOAT_Compiler;
+
+namespace SymbolTableTest
+{
+    public class IntegrationTest
```

Comment:
integration test isnt a fitting name anymore, also the name should relate to symbol table

Reply:
Renamed it to SymbolTableTest